kube-app-testing.sh

```diff
@@ -24,6 +24,13 @@ PRE_TEST_SCRIPT_PATH="ci/pre-test-hook.sh"
 DEFAULT_CLUSTER_TYPE=kind
 PYTHON_TESTS_DIR="test/kat"
 
+# gs cluster config
+DEFAULT_GS_RELEASE="11.2.1"
```

Comment:
I don't intend keeping this as a hard-coded variable - it's just for testing purposes at the moment.